### PR TITLE
add dibi3 support

### PR DESCRIPTION
**Pull request by djdaca**
Jelikož https://github.com/nextras/migrations/pull/39 stále nebyl vyřešen přidal jsem podporu pro dibi3 tak aby stávající podpora pro dibi2 byla zachována.

Dej vědět jestli je to OK.

### CODE REVIEW

**Comment by hrach** [comment]
Generally, I'm ok with this. Just few minor coding-style changes are needed:
empty line at the end, before the class, better commit message :)

**Comment by djdaca** [comment]
ok, is better now ? :)

File: src/Bridges/Dibi/Dibi3Adapter.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * This file is part of the Nextras community extensions of Nette Framework
+ *
+ * @license    New BSD License
+ * @link       https://github.com/nextras/migrations
+ */
+
+
```
Comment (hrach) [inline src/Bridges/Dibi/Dibi3Adapter.php]: here remove one line :-)

File: src/Bridges/Dibi/Dibi3Adapter.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+namespace Nextras\Migrations\Bridges\Dibi;
+
+use DateTime;
+use Dibi\Connection as DibiConnection;
+use Nextras\Migrations\IDbal;
+
+class Dibi3Adapter extends DibiAdapter implements IDbal
```
Comment (hrach) [inline src/Bridges/Dibi/Dibi3Adapter.php]: before this add one line :-)

Comment (hrach) [inline src/Bridges/Dibi/Dibi3Adapter.php]: ok and one last thing, do not extend the original driver, make it indenpendent :-)

**Comment by hrach** [comment]
added few notes :-) at the end, the commits should be squashed. But wait please for @JanTvrdik response :-)

**Comment by djdaca** [comment]
CZ: přizpůsobil jsem to vašemu standartu a udělal třídu nezávislou.

**Comment by djdaca** [comment]
Idea for thinking - in directory Bridges are directories DoctrineDbal and NextrasDbal but i dont understand why not DibiDbal or NetteDatabaseDbal or only Doctrine and Nextras

**Comment by JanTvrdik** [comment]
@djdaca Because the name of the projects we *bridge to* is 

* dibi/dibi -> simplified to dibi
* doctrine/dbal
* nextras/dbal
* nette/database

**Comment by JanTvrdik** [comment]
@djdaca Good job, I solved testing of adapters for both dibi 2 and dibi 3 with some Composer magic and introduced standalone Dibi2Adapter, see #45

**Pull request closed without merging**